nahusenayElias/Team_4
# DRD-21: Create "about us" and "contact" content on Drupal, fetch them on frontend

## Related ticket / Customer approval:
[DRD-21](https://edu-team4-react24k.atlassian.net/browse/DRD-21?atlOrigin=eyJpIjoiMTVjYzVlNGU1Yzg5NDViZDgzNzBjY2FjNWNkN2MwYWIiLCJwIjoiaiJ9)
[DRD-33](https://edu-team4-react24k.atlassian.net/browse/DRD-33?atlOrigin=eyJpIjoiZTFkNDEyYWFmZWQ0NDMxMGI5OTEwNzU5MGUwNGE2NDgiLCJwIjoiaiJ9)

## In this PR:
- Created content-type and content for "contact" in drupal
- Created text content for "about us" in drupal
- Fetch the content on frontend
- Linked "jobs" to Druid's career site

## Testing instructions:
- Checkout branch
- Import db `lando db-import dump.sql.gz`
- Import configs `lando drush cim`
- do `lando drush cr`
- remember to check `services/api.jsx` points to drupal's localhost address
- Check that text content for "about us" and "content" is being fetched on frontend

[DRD-21]: https://edu-team4-react24k.atlassian.net/browse/DRD-21?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DRD-33]: https://edu-team4-react24k.atlassian.net/browse/DRD-33?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
